### ISSUE TITLE
Discussion: Training data selection based on features of the test set

### ISSUE THREAD

**Issue by chrishokamp** [qe-team/marmot#27]
We haven't really covered this topic at all, but it could make a big difference. The WMT14 data has many sentences which don't seem to have any connection to the test set. Removing these instances is likely to improve performance. 

In general, selecting a training set which has good coverage of the features in the test set is a good idea.
